extensions/terminal-suggest/src/tokens.ts

```diff
@@ -87,23 +108,27 @@ export function getTokenType(ctx: { commandLine: string; cursorPosition: number
 	const words = trimmedAfterSeparator.split(/\s+/).filter(w => w.length > 0);
 	
 	// Single complete word followed by more text usually means arguments
+	// Examples: "git status --all|" → Argument, "git s|" → Command (short second word)
```

Comment:
This case is wrong `"git s|" → Command (short second word)`, it should be `-> Argument`. Fix it and add a test.